mappings/net/minecraft/client/render/model/BakedModelManager.mapping

```diff
@@ -1,17 +1,17 @@
 CLASS net/minecraft/class_1092 net/minecraft/client/render/model/BakedModelManager
 	FIELD field_20277 colorMap Lnet/minecraft/class_324;
-	FIELD field_20278 stateToModelIndex Lit/unimi/dsi/fastutil/objects/Object2IntMap;
+	FIELD field_20278 stateLookup Lit/unimi/dsi/fastutil/objects/Object2IntMap;
 	FIELD field_5407 missingModel Lnet/minecraft/class_1087;
-	FIELD field_5408 modelMap Ljava/util/Map;
+	FIELD field_5408 models Ljava/util/Map;
 	FIELD field_5409 spriteAtlas Lnet/minecraft/class_1059;
-	FIELD field_5410 blockStateMaps Lnet/minecraft/class_773;
+	FIELD field_5410 blockModelCache Lnet/minecraft/class_773;
 	METHOD <init> (Lnet/minecraft/class_1059;Lnet/minecraft/class_324;)V
 		ARG 1 spriteAtlas
 		ARG 2 colorMap
-	METHOD method_21611 shouldRerender (Lnet/minecraft/class_2680;Lnet/minecraft/class_2680;)Z
-		ARG 1 old
-		ARG 2 updated
+	METHOD method_21611 statesChanged (Lnet/minecraft/class_2680;Lnet/minecraft/class_2680;)Z
```

Comment:
This returns if the two states have different models rather than if the states "have changed" (too ambiguous imo)

Reply:
Alright. I don't think `shouldRerender` is accurate either. I'll try for something better once I get Enigma booted up.